pq-crypto/bike/decode.c

```diff
@@ -23,7 +23,7 @@
 #include "gf2x.h"
 
 // Decoding (bit-flipping) parameter
-#define MAX_IT 25
+#define MAX_IT 4

```

Comment:
Algorithmic changes require justification. I have started an email thread to get information about the analysis of these changes.